### PR TITLE
Move tuned-adm to builtin repo

### PR DESCRIPTION
**Pull request by linsword13**
Also update the active profile query to check all nodes.

### CODE REVIEW

File: var/ramble/repos/builtin/modifiers/tuned-adm/modifier.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                if "active profile:" in line:
+                    profiles.add(line.split(":")[-1].strip())
+            profiles_str = ",".join(profiles)
+
+        if profiles:
+            with open(profile_path, "a") as f:
+                profiles_str = ",".join(profiles)
+                f.write(f"Applied profiles: {profiles_str}")
+
+    figure_of_merit(
```
Comment (douglasjacobsen) [inline var/ramble/repos/builtin/modifiers/tuned-adm/modifier.py:75]: Odd request, so feel free to push back if it's a bad idea.

Do you think we could have the count of different profiles tracked as a FOM as well?

Comment (linsword13) [reply var/ramble/repos/builtin/modifiers/tuned-adm/modifier.py:75]: No I think this is a good idea, in fact I was thinking maybe it'd be even more direct to have a `success_criteria` that checks there's a single profile and that matches with the given `tuning-profile` value?

Comment (douglasjacobsen) [reply var/ramble/repos/builtin/modifiers/tuned-adm/modifier.py:75]: Yeah, that sounds like a good idea too.

Comment (linsword13) [reply var/ramble/repos/builtin/modifiers/tuned-adm/modifier.py:75]: Done.

**Review by douglasjacobsen — APPROVED** [review]

**Pull request merged**